src/idl_gen_swift.cpp

```diff
@@ -0,0 +1,719 @@
+#include <unordered_set>
+
+#include "flatbuffers/code_generators.h"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/idl.h"
+#include "flatbuffers/util.h"
+
+namespace flatbuffers {
+
+namespace swift {
+
+inline std::string GenIndirect(std::string reading) {
+  return "{{ACCESS}}.indirect(" + reading + ")";
+}
+
+inline std::string GenArrayMainBody(std::string optional = "") {
+  return "\tpublic func {{VALUENAME}}(at index: Int32) -> {{VALUETYPE}}" +
+         optional + " { ";
+}
+
+inline void LowerCase(std::string *value) {
+  transform(value->begin(), value->end(), value->begin(),
+            [](unsigned char c) { return std::tolower(c); });
+}
+
+class SwiftGenerator : public BaseGenerator {
+ private:
+  CodeWriter code_;
+  std::unordered_set<std::string> keywords_;
+
+ public:
+  SwiftGenerator(const Parser &parser, const std::string &path,
+                 const std::string &file_name)
+      : BaseGenerator(parser, path, file_name, "", "."),
+        cur_name_space_(nullptr) {
+    static const char *const keywords[] = {
```

Comment:
The private `should_generate_extension` not inited in ctor.
Why you need this pointer? See my comment on line 704

Reply:
V730 [CWE-457] Not all members of a class are initialized inside the constructor. Consider inspecting: should_generate_extension, namespace_depth. idl_gen_swift.cpp 32